go/vendor/github.com/Keybase/stellarnet/format.go

```diff
@@ -0,0 +1,114 @@
+package stellarnet
```

Comment:
is it a path casing issue?
`go/vendor/github.com/Keybase`
there is upper case `K` in `Keybase`.

I'm just guessing, I didn't try to compile it myself

Reply:
now that's interesting
```
zapu@kyoto ~/…/github.com/Keybase » ls                                                           8:17:59
total 8,2k
drwxrwxr-x 2 zapu zapu 4,1k sty 10 18:31 go-winio/
drwxrwxr-x 2 zapu zapu 4,1k lut 27 08:17 stellarnet/
zapu@kyoto ~/…/github.com/Keybase » ls -R                                                        8:17:59
.:
total 8,2k
drwxrwxr-x 2 zapu zapu 4,1k sty 10 18:31 go-winio/
drwxrwxr-x 2 zapu zapu 4,1k lut 27 08:17 stellarnet/

./go-winio:
total 185k
-rw-rw-r-- 1 zapu zapu 1,1k sty 10 18:31 LICENSE
-rw-rw-r-- 1 zapu zapu 1,1k sty 10 18:31 README.md
-rw-rw-r-- 1 zapu zapu 8,1k sty 10 18:31 backup.go
-rw-rw-r-- 1 zapu zapu 3,2k sty 10 18:31 ea.go
-rw-rw-r-- 1 zapu zapu 7,4k sty 10 18:31 file.go
-rw-rw-r-- 1 zapu zapu 2,2k sty 10 18:31 fileinfo.go
-rw-rw-r-- 1 zapu zapu  12k sty 10 18:31 pipe.go
-rw-rw-r-- 1 zapu zapu 5,5k sty 10 18:31 privilege.go
-rw-rw-r-- 1 zapu zapu 3,5k sty 10 18:31 reparse.go
-rw-rw-r-- 1 zapu zapu 3,3k sty 10 18:31 sd.go
-rw-rw-r-- 1 zapu zapu  162 sty 10 18:31 syscall.go
-rw-rw-r-- 1 zapu zapu  18k sty 10 18:31 zsyscall_windows.go

./stellarnet:
total 13k
-rw-rw-r-- 1 zapu zapu 3,0k lut 27 08:17 format.go
```

Reply:
where in the world did it get a capitalized Keybase from?  locally it is not capitalized for me in this branch (or anywhere).

Reply:
looks like this Windows thing got there before which did not matter on Windows (because of uncased file systems), and somehow govendor got the wrong directory when you were revendoring? Unless you copied that file manually and picked wrong directory.

Reply:
I used govendor.

Reply:
now that's worrying. I wonder if we can repro this